src/math-window.c

```diff
@@ -329,14 +330,17 @@ static void about_cb(GtkWidget* widget, MathWindow* window)
     for (p = documenters; *p; ++p)
         *p = _(*p);
 
+    comments = g_strdup_printf (_("Calculator with financial and scientific modes.\nUsing GNU MPFR %s and GNU MPC %s"),
```

Comment:
full stop is missing at the end of the sentence
```patch
diff --git a/src/math-window.c b/src/math-window.c
index 87ac779..d30b784 100644
--- a/src/math-window.c
+++ b/src/math-window.c
@@ -330,7 +330,7 @@ static void about_cb(GtkWidget* widget, MathWindow* window)
     for (p = documenters; *p; ++p)
         *p = _(*p);
 
-    comments = g_strdup_printf (_("Calculator with financial and scientific modes.\nUsing GNU MPFR %s and GNU MPC %s"),
+    comments = g_strdup_printf (_("Calculator with financial and scientific modes.\nUsing GNU MPFR %s and GNU MPC %s."),
                                 mpfr_get_version (), mpc_get_version ());
 
     gtk_show_about_dialog(GTK_WINDOW(window),
```

Reply:
it looks weird for me with the dot at the end next to the version

mate-terminal commit with VTE version doesn't have it

Reply:
PR https://github.com/mate-desktop/mate-terminal/pull/338